### PR TITLE
Exchange rate system contract

### PR DESCRIPTION
**Pull request by tinker-michaelj**
**Description**:
 - Proposes a new system contract at `0x168` that converts between tinycents and tinybars at the network's active exchange rate in system file `0.0.112`.

### CODE REVIEW

File: hip-0000-exchange-rate-system-contract.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+created: 05-12-2022
+discussions-to: https://github.com/hashgraph/hedera-improvement-proposal/discussions/474
+---
+
+## Abstract
+
+We propose a new precompiled contract (or _system contract_) to convert between tinybars and tinycents at the network's 
+active exchange rate in system file `0.0.112`; that is, at the exchange rate the network is using to calculate fees. 
+
+This precompile would **NOT** be a live price oracle appropriate to use in, for example, a DEX client contract.
```
Comment (shemnon) [inline hip-0000-exchange-rate-system-contract.md:19]: I might include some verbiage about how this exchange rate is used to convert from the fee schedule to fees in tinybars.

File: hip-0000-exchange-rate-system-contract.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+}
+```
+
+## Backwards Compatibility
+
+There are no known sources of backward incompatibility.
+
+## Security Implications
+
+There are no obvious security implications. Even if the exchange rate in file `0.0.112` diverges from the USD-HBAR exchange
```
Comment (shemnon) [inline hip-0000-exchange-rate-system-contract.md:78]: "There are no obvious security implications." famous last words :)

I think we need to enumerate the risk of an exchange depending on this precompile as an exchange oracle.  Even though we call that out as a bad idea we need to explain potential impacts and mitigations. Exposing this info to smart contracts is a new risk surface.

**Review by shemnon — APPROVED** [review]

**Review by mgarbs — APPROVED** [review]
Looks good to start tracking!

File: hip-0000-exchange-rate-system-contract.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+category: Service
+needs-council-approval: Yes
+status: Draft
+created: 05-12-2022
+discussions-to: https://github.com/hashgraph/hedera-improvement-proposal/discussions/474
+---
+
+## Abstract
+
+We propose a new precompiled contract (or _system contract_) to convert between tinybars and tinycents at the network's
```
[the remark this answers is not in the document]
Comment (tinker-michaelj) [reply hip-0000-exchange-rate-system-contract.md]: Updated wording.

File: hip-0000-exchange-rate-system-contract.md (hunk 2 above)
Comment (tinker-michaelj) [reply hip-0000-exchange-rate-system-contract.md]: Highlighted this risk.

**Review by mgarbs — DISMISSED** [review]
LGTM

**Review by mgarbs — APPROVED** [review]
LGTM

**Pull request merged**